pymysql/converters.py

```diff
@@ -309,13 +309,8 @@ def through(x):
     return x
 
 
-#def convert_bit(b):
-#    b = "\x00" * (8 - len(b)) + b # pad w/ zeroes
-#    return struct.unpack(">Q", b)[0]
-#    
-#     the snippet above is right, but MySQLdb doesn't process bits,
-#     so we shouldn't either
```

Comment:
MySQL-python (aka, MySQLdb) author is inactive for these years.
I maintains fork of it named mysqlclient-python.
But I won't break backward compatibility until I am official maintainer of MySQLdb.